bin/v-stop-firewall

```diff
@@ -80,7 +80,7 @@ else
     preup="/etc/network/if-pre-up.d/hestia-iptables"
     # Recreate the Hestia iptables rules loading script
     rm -f $routable $preup
-    if dpkg-query -W -f'${Status}' "netplan*" 2>/dev/null | grep -q "ok installed" && [ -d /etc/netplan ] && [ -n "$(ls -A /etc/netplan 2>/dev/null)" ]; then
+    if dpkg-query -W -f'${Status}' "netplan.io" 2>/dev/null | grep -q "ok installed" && [ -n "$(ls -A /etc/netplan 2>/dev/null)" ] && systemctl is-enabled networkd-dispatcher >/dev/null 2>&1; then
```

Comment:
Still have issues with Ubuntu 20.04 + Systemd  with the code chages. It seems that 

/etc/network/if-pre-up.d/ folder does not exists on default and creating the folder doesn't seem to help.

What did help was creating an systemd service:

```
[Unit]
Description=iptables firewall service
After=network.target

[Service]
Type=oneshot
ExecStart=/usr/local/hestia/bin/v-update-firewall 
RemainAfterExit=true

[Install]
WantedBy=multi-user.target
```
We should probably implement something like that for the cases. Might even work find in the other cases .

Reply:
Directly executing "v-update-firewall" when starting system will cause potential conflicts, such as Docker and WireGuard, etc.

A more reasonable solution is to create a special loading script for Proxmox or OpenVZ, and then Systemd will execute it when starting system.

Reply:
I'm going to bed, and I will try to make a pure Systemd solution tomorrow.

Reply:
👍🏼

Reply:
@jaapmarcus The Systemd service is ready for testing.

But I don't have Proxmox (You mean LXC, right?) or OpenVZ VM, so I need your help to test it.

Follow the steps:

1. Delete the legacy version loading script
And don't run "v-stop-firewall/v-update-firewall" before restart system
```
rm -f /usr/lib/networkd-dispatcher/routable.d/10-hestia-iptables /etc/network/if-pre-up.d/hestia-iptables
```

2. Paste all lines to create the service
```
cat << "EOF" > /lib/systemd/system/hestia-iptables.service
[Unit]
Description=Loading Hestia firewall rules
DefaultDependencies=no
Wants=network-pre.target local-fs.target
Before=network-pre.target
After=local-fs.target

[Service]
Type=oneshot
RemainAfterExit=yes
ExecStartPre=-/usr/local/hestia/bin/v-update-firewall-ipset
ExecStart=/sbin/iptables-restore /etc/iptables.rules

[Install]
WantedBy=multi-user.target
EOF
```

3. Enable the service and restart system
```
systemctl enable hestia-iptables
systemctl reboot
```

4. Check iptables status in Web-UI